Makefile

```diff
@@ -5,4 +5,13 @@ lua:
 	mv A6/ lua/
 
 .PHONY: compiled
-compiled: lua
+
+.PHONY: java
+java:
+	flatc --java ext-plugin.fbs
+	rm -rf java/io.github.api7.A6
+	mv A6/ java/io.github.api7.A6/
+	find . -name "*.java" | xargs sed -i 's/A6/io.github.api7.A6/g'
+
+.PHONY: compiled
+compiled: lua java
```

Comment:
Need to add the `java` directory to `.gitignore`.

Reply:
I see that the lua code generated by flatbuffers is not in the   `.gitignore`, so I thought the java ones didn't need. @spacewander what do you think?